apps/dashboard/src/components/subscribers/create-subscriber-form.tsx

```diff
@@ -104,7 +104,12 @@ export const CreateSubscriberForm = (props: CreateSubscriberFormProps) => {
         </div>
       </header>
       <Form {...form}>
-        <FormRoot autoComplete="off" noValidate onSubmit={form.handleSubmit(onSubmit)} className="flex h-full flex-col">
+        <FormRoot
+          autoComplete="off"
+          noValidate
+          onSubmit={form.handleSubmit(onSubmit)}
+          className="flex h-full flex-col overflow-y-auto"
+        >
```

Comment:
in the create workflow sidebar and step conditions, the create button at the bottom is always visible and only the body scrolls

Reply:
yeah you're right the footer stays visible always in the workflow page
i replicated the behaviour it was on the edit subscriber drawer
will update the create and edit subscriber pages to have the footer always visible